portfolio/src/main/webapp/script.js

```diff
@@ -92,3 +92,12 @@ window.addEventListener('scroll', function() {
     document.getElementById('logo').style.fontSize = '35px';
   }
 })
+
+/**
+ * Fetches a message from the server and adds it to the DOM.
```

Comment:
It would probably be good to note what this message actually is here in addition to the server so it's more accessible to people reading your code!

Reply:
Acknowledged!